client/src/catalog-app.html

```diff
@@ -70,9 +70,13 @@
         }
       },
 
+      isStaging: function() {
```

Comment:
Maybe we should just invert all this and use `!isProduction()`. Don't really need to distinguish between staging/localhost then.